docs/reference/data-streams.md

```diff
@@ -0,0 +1,112 @@
+
+This is the default mode. When ingesting standard OTel signals, EDOT writes to:
+
+| Signal | Data stream |
+|--------|-------------|
+| Traces | `traces-otel-*` |
+| Metrics | `metrics-otel-*` (TSDB-backed using `mode: time_series`) |
+| Logs | `logs-otel-*` |
+| Aggregated metrics | `metrics-*.[1m\|10m\|60m].otel-*` |
+
+These streams follow OTel naming conventions and field structures.
+
+### ECS-compatible mode
+
+When the `ecs` exporter is enabled, or when ingesting data from {{product.apm}} agents using `elasticapmintake`, EDOT writes to:
+
+- `traces-apm-*`
+- `metrics-apm.internal-*`
+- `metrics-apm.*-*`
+- `metrics-apm.*.[interval]-*`
+- `logs-apm.error-*`
+- `logs-apm.app.*-*`
+
+These match legacy {{product.apm-server}} behavior.
+
+## Managed OTLP Endpoint
+
+The {{motlp}} follows the same stream selection logic as local EDOT deployments:
+
+- OTel-native streams for standard OTel telemetry  
+- ECS-compatible streams for {{product.apm}}-formatted events  
+
+The main difference is that {{motlp}} is a managed cloud service that handles ingestion and storage, whereas local EDOT deployments require you to manage the Collector infrastructure yourself. Ingest and storage behavior remain consistent between both approaches.
+
+To customize which data streams your telemetry is routed to, you can use [data stream routing](docs-content://solutions/observability/apm/opentelemetry/data-stream-routing.md) with `data_stream.dataset` and `data_stream.namespace` attributes.
+
+## Field duplication
+
+You may notice fields like:
+
+- `resource.attributes.service.name`
+- `service.name`
+
+These are not true duplicates. Here's why:
+
+- OTel defines attributes at multiple levels (for example: resource, span).
+- EDOT stores resource attributes under `resource.attributes.*`, according to OTel specifications.
+- {{kib}} dashboards and {{product.apm}} UI rely on ECS-style top-level fields like `service.name`.
+
+For more information about how resource attributes are mapped to ECS fields and stored, refer to [Attributes and labels](docs-content://solutions/observability/apm/opentelemetry/attributes.md).
+
+EDOT uses two mechanisms to bridge these models:
+
+1. **Passthrough fields**: When fields are propagated from `resource.attributes` to top level (also for `scope.attributes` and `attributes`), EDOT uses passthrough fields with no storage overhead.
+
+2. **Field copying**: EDOT performs some copying for specific fields for enrichment and compatibility purposes. This copying is separate from the passthrough mechanism.
+
+These mechanisms ensure that:
+
+- Dashboards continue to function  
+- Filters and searches behave consistently  
+- OTel structure remains intact for downstream compatibility  
+
+This duplication is intentional.
+
+## Storage engines used by EDOT
+
+| Data type | Storage engine |
+|-----------|----------------|
+| Metrics   | TSDB (`mode: time_series`) |
+| Logs      | LogsDB |
+| Traces    | Standard time-series model |
```

Comment:
I'm not sure what's the audience of this doc, but mapping EDOT to storage engine is not that simple. EDOT is responsible for sending to a data stream, and depending on which data stream it sends to, different index templates may apply, and the index templates decide what storage engine is used.

e.g. EDOT with otel mapping mode sends to otel data streams, and the table is correct. But EDOT with ecs mapping mode sending to apm data streams, the table will be incorrect.